src/ralph/api/forwarding.py

```diff
@@ -32,13 +32,16 @@ def get_active_xapi_forwardings() -> List[XapiForwardingConfigurationSettings]:
     return active_forwardings
 
 
-async def forward_xapi_statements(statements: List[dict]):
+async def forward_xapi_statements(
+    statements: List[dict], method: Literal["post", "put"]
```

Comment:
Would it work if we call this function with a list of statements and selecting the "put" method?
We could probably accept either a list of statements for POST or a single statement for PUT. WDYT?

Reply:
This was changed to:

```async def forward_xapi_statements(statements: Union[dict, List[dict]], method: Literal["post", "put"]):```